RedHat-UX/red-hat-design-system
# fix(secondary-nav): font-size initial, logo font-size, and cta css property

## What I did

1. Removed `font-size: initial` from lightdom CSS, added correctly to `rh-secondary-nav`   on `:host`.  
2. Set `font-size: 1.125 !important` for slotted logo on `rh-secondary-nav`.
3. Set `--rh-font-size-text-xl: 1em !important` for slotted cta's on both `rh-secondary-nav` and `rh-secondary-nav-menu-section`.
4. Closes #589 


## Testing Instructions

1. Checkout branch
2. Uncomment `D7` comments in `rh-secondary-nav/demo/demo.css`
3. Run `npm start`
4. Should see no change from pre and post-upgrade CSS for menu and no FOUC.

## Notes to Reviewers
You will need to uncomment  the styles in the `demo.css` and run locally or add the following styles to the deploy preview via inspector:  

```css
html {
  font-size: 62.5%;
} 
body { 
  font-size: 1.6rem;
}
```
